### PR TITLE
Run as slack user

### PR DESCRIPTION
**Pull request by anojansivarajah**
Implemented for Rundeck driver

### CODE REVIEW

**Comment by anojansivarajah** [comment]
@outofcoffee Is this okay? when are you planning to merge develop to master?

**Comment by anojansivarajah** [comment]
@pcornish @outofcoffee Any updates?

File: drivers/rundeck/src/main/kotlin/com/gatehill/corebot/driver/rundeck/action/RundeckJobTriggerService.kt [hunk 1]
```diff
@@ -37,7 +37,7 @@ class RundeckJobTriggerService @Inject constructor(private val actionDriver: Run
         try {
             call = actionDriver.buildApiClient().runJob(
                     jobId = action.jobId,
-                    executionOptions = ExecutionOptions(argString = buildArgString(args))
+                    executionOptions = ExecutionOptions(argString = buildArgString(args), asUser = trigger.username)
```
Comment (outofcoffee) [inline drivers/rundeck/src/main/kotlin/com/gatehill/corebot/driver/rundeck/action/RundeckJobTriggerService.kt]: Looks like this branch might be missing the changes to `RundeckApi.kt`.

Comment (anojansivarajah) [inline drivers/rundeck/src/main/kotlin/com/gatehill/corebot/driver/rundeck/action/RundeckJobTriggerService.kt]: yes @outofcoffee. asUser is passed in executionoptions. so no need to amend RundeckApi.kt

Comment (anojansivarajah) [inline drivers/rundeck/src/main/kotlin/com/gatehill/corebot/driver/rundeck/action/RundeckJobTriggerService.kt]: @pcornish it build for me and i am able to run some tests

**Pull request merged**